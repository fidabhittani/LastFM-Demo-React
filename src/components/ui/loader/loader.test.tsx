import React from "react";
import { shallow } from "enzyme";
import Loader from "./index";

/** Loader TEST */
describe("Loader Root Component Tests", () => {
  it("Loader Component renders correctly", () => {
    const langWrapper = shallow(<Loader />);
    expect(langWrapper).toMatchSnapshot();
  });
});
