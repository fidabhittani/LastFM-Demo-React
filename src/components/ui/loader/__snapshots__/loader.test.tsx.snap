// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Root Component Tests Loader Component renders correctly 1`] = `ShallowWrapper {}`;
